web_console/customizing-the-web-console.adoc

```diff
@@ -17,6 +17,10 @@ include::modules/creating-custom-links.adoc[leveloffset=+1]
 
 include::modules/customizing-the-web-console-URL.adoc[leveloffset=+1]
 
+== Recognizing resource and project limits and quotas
+
+A message appears in the *Topology* or *List* views accessible through the web console, when a deployment reaches its preset limits or quotas. 
+
```

Comment:
```suggestion
[NOTE]
====
When a deployment reaches its preset limits or quotas, a message appears in the *Topology* and *List* views of the web console.
====

```

Reply:
Left as a separate section because I increased the amount of content to resolve other comments.